packages/documentation-site/patternfly-docs/content/demos/Compass/CompassIntegrationsDemo.tsx

```diff
@@ -0,0 +1,604 @@
+    setIsThinking(true);
+    setTimeout(() => {
+      setIsThinking(false);
+    }, 10000); // 10 seconds
+  };
+
+  const navContent = (
+    <>
+      <CompassPanel isPill hasNoPadding>
+        <Tabs
+          activeKey={1}
+          isNav
+          onSelect={() => {}}
+          component={TabsComponent.nav}
+          aria-label="Compass navigation tabs"
+          inset={{ default: 'insetXl' }}
+        >
+          <Tab eventKey={0} title={<TabTitleText>Tab 1</TabTitleText>} />
+          <Tab eventKey={1} title={<TabTitleText>Integrations</TabTitleText>} />
+          <Tab eventKey={2} title={<TabTitleText>Tab 3</TabTitleText>} />
+          <Tab eventKey={3} title={<TabTitleText>Disabled Tab 4</TabTitleText>} isDisabled />
+        </Tabs>
+      </CompassPanel>
+    </>
+  );
+
+  const sidebarContent = (
+    <CompassPanel isPill>
+      <ActionList isIconList isVertical>
+        <ActionListGroup>
+          <ActionListItem>
+            <Tooltip content="Play">
+              <Button isCircle variant="plain" icon={<PlayIcon />} aria-label="Play" />
+            </Tooltip>
+          </ActionListItem>
+          <ActionListItem>
+            <Tooltip content="Add">
+              <Button isCircle variant="plain" icon={<OutlinedPlusSquare />} aria-label="Add" />
+            </Tooltip>
+          </ActionListItem>
+        </ActionListGroup>
+        <ActionListItem>
+          <Tooltip content="Copy">
+            <Button className="pf-v6-m-ai-indicator" isCircle variant="plain" icon={<OutlinedCopy />} aria-label="Copy" />
+          </Tooltip>
+        </ActionListItem>
+        <ActionListGroup>
+          <ActionListItem>
+            <Tooltip content="Help">
+              <Button isCircle variant="plain" icon={<OutlinedQuestionCircleIcon />} aria-label="Help" />
+            </Tooltip>
+          </ActionListItem>
+          <ActionListItem>
+            <Tooltip content="Second copy">
+              <Button isCircle variant="plain" icon={<OutlinedCopy />} aria-label="Copy2" />
+            </Tooltip>
+          </ActionListItem>
+        </ActionListGroup>
+      </ActionList>
+    </CompassPanel>
+  );
+
+  const userDropdownItems = (
+    <>
+      <DropdownItem>My profile</DropdownItem>
+      <DropdownItem>User management</DropdownItem>
+      <DropdownItem>Logout</DropdownItem>
+    </>
+  );
+
+  const userDropdown = (
+    <Dropdown
+    isOpen={isDropdownOpen}
+    onSelect={() => setIsDropdownOpen(false)}
+    onOpenChange={(isOpen: boolean) => setIsDropdownOpen(isOpen)}
+    popperProps={{ position: "right" }}
+    toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+      <MenuToggle
+        ref={toggleRef}
+        onClick={() => setIsDropdownOpen(!isDropdownOpen)}
+        isExpanded={isDropdownOpen}
+        variant="plain"
+        isCircle
+      >
+        <Flex
+          alignItems={{ default: "alignItemsCenter" }}
+          gap={{ default: "gapMd" }}
+        >
+          User Name
```

Comment:
I'm using the name from the POC here, not that it really matters what we use but I think we should probably standardize on one for consistency